mllib-local/src/test/scala/org/apache/spark/ml/util/TestingUtils.scala

```diff
@@ -48,7 +48,7 @@ object TestingUtils {
   /**
    * Private helper function for comparing two values using absolute tolerance.
    */
-  private def AbsoluteErrorComparison(x: Double, y: Double, eps: Double): Boolean = {
+  private[ml] def AbsoluteErrorComparison(x: Double, y: Double, eps: Double): Boolean = {
```

Comment:
It's not used. I just changed the scope of both methods, I can change it back of course. I don't see a great reason to make this public since most users will use `relTol` instead. I'm open to other opinions though.